### ISSUE TITLE
Adding timestap to the logging of the commands in the build script

### ISSUE THREAD

**Issue by CristhianMotoche** [stackbuilders/hapistrano#74]
Adding timestap to the log will help to know when exactly a command was executed. The timestapms could be like the following example from capistrano.
```
[2017-07-21T16:50:56.924828 #19038]  INFO -- : Writing /var/projects/projecName/assets/img/picture
```

**Comment by po5i** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Adding timestap to the logging of the commands in the build script](https://trello.com/c/wJ4EzNQl/49-adding-timestap-to-the-logging-of-the-commands-in-the-build-script)

**Comment by CristhianMotoche** [comment]
Solved in #111
